openeo_processes_dask/process_implementations/cubes/_filter.py

```diff
@@ -88,3 +91,69 @@ def filter_bands(data: RasterCube, bands: list[str] = None) -> RasterCube:
             f"The provided bands: {bands} are not all available in the datacube. Please modify the bands parameter of filter_bands and choose among: {data[band_dim].values}."
         )
     return data
+
+
+def filter_bbox(data: RasterCube, extent: BoundingBox) -> RasterCube:
+    try:
+        input_crs = str(data.rio.crs)
+    except Exception as e:
+        raise Exception(f"Not possible to estimate the input data projection! {e}")
+    trasnformed_extent = reproject_bbox(extent, input_crs)
+
+    # Check if the coordinates are increasing or decreasing
+    if len(data.y) > 1:
+        if data.y[0] > data.y[1]:
+            y_slice = slice(trasnformed_extent.north, trasnformed_extent.south)
+        else:
+            y_slice = slice(trasnformed_extent.south, trasnformed_extent.north)
+    if len(data.x) > 1:
+        if data.x[0] > data.x[1]:
+            x_slice = slice(trasnformed_extent.east, trasnformed_extent.west)
+        else:
+            x_slice = slice(trasnformed_extent.west, trasnformed_extent.east)
+
+    aoi = data.loc[{"y": y_slice, "x": x_slice}]
+    return aoi
+
+
+def reproject_bbox(extent: BoundingBox, output_crs: str) -> BoundingBox:
```

Comment:
```suggestion
def _reproject_bbox(extent: BoundingBox, output_crs: str) -> BoundingBox:
```
Since this is a private method

Reply:
Also I'd call this `target_crs` instead!